oc_cdtapi/BitbucketAPI.py

```
@@ -0,0 +1,200 @@
+        logging.debug('Archived repository: %s', repo_data.get('name', 'unknown'))
+        
+        return repo_data
+    
+    def get_repos(self, project):
+        """
+        Get list of repositories for a project
+        :param str project: project key
+        :return list: repositories
+        """
+        logging.debug('Reached %s.get_repos', self.__class__.__name__)
+        logging.debug('project: {0}'.format(project))
```

Ruff (0.15.12)

[warning] Use implicit references for positional format fields

Remove explicit positional indices

(UP030)